sympy/utilities/lambdify.py

```diff
@@ -386,6 +386,14 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     namespace.update({'builtins':builtins, 'range':range})
 
     func = eval(lstr, namespace)
+    # For numpy lambdify, wrap all input arguments in arrays.
+    # This is a fix for gh-11306.
+    if module_provided and _module_present('numpy',namespaces):
```

Comment:
a whitespace is missing before namespaces. Don't we run pep8 on all modules to catch such coding style violations?